net.hachune.mikutter.yaml

```diff
@@ -0,0 +1,69 @@
+app-id: net.hachune.mikutter
+runtime: org.gnome.Platform
+runtime-version: 3.28
+sdk: org.gnome.Sdk
+command: mikutter
+separate-locales: false
+finish-args:
+  # X11 + XShm access
+- --share=ipc
+- --socket=x11
+  # Wayland access
+- --socket=wayland
+- --socket=pulseaudio
+  # Notification access
+- --talk-name=org.freedesktop.Notifications
+- --share=network
+- --filesystem=home
```

Comment:
Wouldn't xdg-pictures be good enough?

Reply:
mikutter need to access ~/.mikutter and this line should be `--filesystem=~/.mikutter`, however the gtk theme and font setting is not applied without `--filesystem=home`. Is it the flatpak spec or a bug?

Reply:
No, it should be `--persit=.mikutter` or to be perfectly correct, start following XDG specs and use files from `$XDG_CONFIG_DIR/.mikutter` (which is `~/.config` by default). If it's GTK3 app, theme should be packaged separately and put on Flathub.

Reply:
Thank you for your advice.

> start following XDG specs

migration to path following XDG specs is under discussion.

>  the gtk theme and font setting is not applied without `--filesystem=home`

It was just my environment problem (installing theme package and switching to wayland session solved it).